scripts/deps/config/besu/genesis.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
IMO we should be using the same `genesis.go` we have in Orchestrate for a better compatibility. Let me know if you need me to point you to it

Reply:
Yes please

Reply:
https://gitlab.com/ConsenSys/client/fr/core-stack/orchestrate/-/tree/master/scripts/besu/config

Reply:
The genesis you point to in Orchestrate is an IBFT2 one. I want a Clique in this QS as it works with only one node and also we don’t need privacy feature or P+ for this quickstart. The Clique extra-data part is also easier to setup and totally enough for this kind of use.